### ISSUE TITLE
using the theme on an iphone 5

### ISSUE THREAD

**Issue by joergbirkhold** [blackberry/jQueryMobile-BB10-Theme#133]
hey,
maybe this is not the right place to ask such a question but:
I tried to use the theme on an iphone 5 and everything is much too big. 
can I somehow adjust this?

**Comment by mdabydeen** [comment]
@joergbirkhold I don't think this theme is designed to work on the iPhone 5. Have you tried using the [jQuery Mobile](http://jquerymobile.com/)?
